modules/Search/components/Hit.tsx

```diff
@@ -36,6 +36,7 @@ export function Hit({ hit }: Props) {
                     name,
                     slug,
                     description: null,
+                    public_stories_number: 0, // We do not have access to number of stories from Algolia
```

Comment:
We don't really need this property in this code, so just adding it here to satisfy the type.